os/pm/pm_procfs.c

```diff
@@ -118,6 +118,7 @@ struct power_procfs_entry_s {
 
 /* Added to record timer countdown interrupt */
 struct pm_wakeup_timer_s g_timer_wakeup = { 0, 0 };
+struct pm_wakeup_timer_s g_timer_lock = { 0, 0 };
```

Comment:
This might be redundant, we can expand the definition in pm_wakeup_timer_s, to handle more kind of utilizations.